modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1839,7 +1837,7 @@ private TableImpl createTableImpl(
                 () -> txCfg.value().readWriteTimeoutMillis(),
                 () -> txCfg.value().readOnlyTimeoutMillis(),
                 nodeProperties.colocationEnabled(),
-                createAndRegisterMetricsSource(tableStorage.getTableDescriptor(), tableName)
+                tableMetrics.createAndRegisterMetricsSource(tableStorage.getTableDescriptor(), tableName, null, null)
```

Comment:
IMHO, this approach does not work. We need to change `TableMetricSource` here after renaming. Holding an internal `holder` is not enough, I think.
By the way, I already have a ticket to address this issue https://issues.apache.org/jira/browse/IGNITE-26524